locales/es/messages.po

```diff
@@ -593,10 +593,6 @@ msgstr "Estimado dueño/manager del edificio."
 msgid "Decide how to send your letter"
 msgstr "Decide how to send your letter"
 
-#: common-data/issue-choices-laletterbuilder.ts:23
-msgid "Defective electricity"
-msgstr "Defective electricity"
-
```

Comment:
Any idea why these are being removed? no biggie just curious

Reply:
it seems like they were duplicates of the ones below? this might have been a problem i created when i incorrectly merged a translation PR from crowdin. in order to get crowdin to see that something needs translation, is the msgstr supposed to be the same in english and spanish, or blank?

Reply:
oh hmmm, I believe in the english file the `msgstr` is the same as the `msgid` (unless a custom id is specified) and then `msgstr` is blank in the spanish file until we actually translate the string in crowdin.